pkg/cri/resource-manager/cache/container.go

```diff
@@ -319,6 +319,17 @@ func (c *container) GetResmgrLabel(key string) (string, bool) {
 	return value, ok
 }
 
+func (c *container) GetLabels() map[string]string {
+	if c.Labels == nil {
+		return nil
+	}
+	labels := make(map[string]string)
```

Comment:
nit: you could use `len(c.Labels)` as the size?

Reply:
Fixed.